src/app/service-file.service.ts

```diff
@@ -12,16 +12,37 @@ export class ServiceFileService {
   constructor(private http: HttpClient) { }
   getPosts() {
     return this.http.get(this.url)
+    .pipe(
+      catchError(this.handleError)
+    )
   }
-  deleteRequest(id: any): Observable<any> {
-    return this.http.delete(`${this.url}/${id}`);
+  deleteRequest(id: number): Observable<any> {
```

Comment:
any should not be used anywhere

Reply:
Provided a type to the variables instead of type "any". And pushed the changes.